src/mykrobe/cmds/amr.py

```diff
@@ -339,17 +345,21 @@ def run(parser, args):
             args.expected_error_rate = 0.15
             args.ploidy = "haploid"
             args.ignore_minor_calls = True
-            logger.warning("Setting ploidy to haploid")
-            logger.warning("Setting ignore_minor_calls to True")
-            logger.warning(
-                "Setting expected error rate to %s (--ont)" % args.expected_error_rate
-            )
+            logger.warning("Setting ploidy to haploid (because --ont flag used)")
```

Comment:
I wouldn't say this is a warning, it's more info or debug surely?

Reply:
I don't really care either way, I just stuck with the level that was already there. It's a warning if it overrides what the user chose. Otherwise it's info.